api/rest/v1/bulk_triples.md

```diff
@@ -0,0 +1,439 @@
+---
+layout: default
+title: Triples
+parent: v1 REST
+grand_parent: API
+nav_exclude: true
+published: false
+permalink: /api/rest/v1/bulk/triples
+---
+
+# /v1/bulk/triples
+
+Get [triples](/glossary.html#triple) for multiple entities.
```

Comment:
please change "entities" to "nodes" throughout

Reply:
Done!